config/toolchain/clang.cmake

```diff
@@ -9,6 +9,7 @@ if(WIN32)
 else()
   set (CMAKE_C_COMPILER clang)
   set (CMAKE_CXX_COMPILER clang++)
+  #set (CMAKE_Fortran_COMPILER flang-new)
```

Comment:
Probably remove?

Reply:
Maybe - maybe not. Thought it was worthwhile to have a file with a one char change to enable it.
Can be persuaded either way.